apps/new/widget/components/project/StarProject.jsx

```diff
@@ -0,0 +1,41 @@
+const { Button } = VM.require("${alias_old}/widget/components") || {
```

Comment:
Let's reuse the logic from [mob.near/widget/StarButton](https://near.social/mob.near/widget/WidgetSource?src=mob.near/widget/N.StarButton)

Reply:
I was going to but it wasn't working for me. I'll try again and let you know.